kafka-streams/src/main/java/io/confluent/developer/aggregate/StreamsAggregate.java

```diff
@@ -31,16 +31,24 @@ public static void main(String[] args) throws IOException {
         final SpecificAvroSerde<ElectronicOrder> electronicSerde =
                 StreamsUtils.getSpecificAvroSerde(configMap);
 
-        final KStream<String, ElectronicOrder> electronicStream =
-                builder.stream(inputTopic, Consumed.with(Serdes.String(), electronicSerde))
-                        .peek((key, value) -> System.out.println("Incoming record - key " +key +" value " + value));
-
-        electronicStream.groupByKey().aggregate(() -> 0.0,
-                                                (key, order, total) -> total + order.getPrice(),
-                                                 Materialized.with(Serdes.String(), Serdes.Double()))
-                                                .toStream()
-                .peek((key, value) -> System.out.println("Outgoing record - key " +key +" value " + value))
-                .to(outputTopic, Produced.with(Serdes.String(), Serdes.Double()));
+        final KStream<String, ElectronicOrder> electronicStream = null;
+              // Using the StreamsBuilder from above, create a KStream with an input-topic
+              // and a Consumed instance with the correct
+              // Serdes for the key, Serdes.String() and the value electronicSerde created above
+
+              // To view the key-value records coming into the application consider
+              //  adding this statement to the KStream .peek((key, value) -> System.out.println("Incoming record - key " +key +" value " + value));
+
+              // Now take the electronicStream object, group by key and perform an aggregation
+              // Don't forget to convert the KTable returned by the aggregate call back to a KStream using the toStream method
+              electronicStream.groupByKey().aggregate(null, null);
+
+              // To view the results of the aggregation consider
+              // right after the toStream() method .peek((key, value) -> System.out.println("Outgoing record - key " +key +" value " + value))
+
+              // Finally write the results to an output topic
+              // using a KStream.to method with a Produced config object
+              // and the appropriate Serdes for the key Serdes.String()  and the value Serdes.Long()
```

Comment:
This is how the exercises will look now, just a few stubs and comments on what's needed to add to make it functional.